Kickstarter-iOS/Views/Controllers/BackerDashboardProjectsViewControllerTests.swift

```diff
@@ -0,0 +1,109 @@
+import Prelude
+import Result
+import XCTest
+@testable import Kickstarter_Framework
+@testable import KsApi
+@testable import Library
+
+// swiftlint:disable:next type_name
+internal final class BackerDashboardProjectsViewControllerTests: TestCase {
```

Comment:
You know me so well. Already did it!

> . It also aids in reviewing pull-requests since the implementation and test files will alternate instead of having them lumped together.